File: src/main/java/com/gimranov/zandy/app/model/Links_.java

package com.gimranov.zandy.app.model;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Links_ implements Serializable
{

    @SerializedName("self")
    @Expose
    public Self self;
    @SerializedName("alternate")
    @Expose
    public Alternate_ alternate;
    private final static long serialVersionUID = -4626930828623708887L;

}

File: src/main/java/com/gimranov/zandy/app/model/Links.kt
package com.gimranov.zandy.app.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Links : Serializable {

    @SerializedName("alternate")
    @Expose
    var alternate: Alternate? = null

    companion object {
        private const val serialVersionUID = 1909908181802478791L
    }

}
